packages/@aws-cdk/aws-rds/lib/cluster-ref.ts

```diff
@@ -46,38 +46,50 @@ export interface IDatabaseCluster extends IResource, ec2.IConnectable, secretsma
  */
 export interface DatabaseClusterAttributes {
   /**
-   * The database port
+   * Identifier for the cluster
    */
-  readonly port: number;
+  readonly clusterIdentifier: string;
 
   /**
-   * The security groups of the database cluster
+   * The database port
+   *
+   * @default - none
    */
-  readonly securityGroups: ec2.ISecurityGroup[];
+  readonly port?: number;
 
   /**
-   * Identifier for the cluster
+   * The security groups of the database cluster
+   *
+   * @default - no security groups
    */
-  readonly clusterIdentifier: string;
+  readonly securityGroups?: ec2.ISecurityGroup[];
 
   /**
    * Identifier for the instances
+   *
+   * @default - no instance identifiers
    */
-  readonly instanceIdentifiers: string[];
+  readonly instanceIdentifiers?: string[];
```

Comment:
was this why we went with `clusterIdentifier` for the property name, or was there another reason?